types/v1/policies.go

```diff
@@ -134,6 +134,10 @@ const (
 
 	// VolumeTypeVK is the key to fetch the volume type
 	VolumeTypeVK VolumeKey = "openebs.io/volume-type"
+
+	// ControllerContainerStatus is the key to fetch the status of
+	// containers in a pod
+	ControllerContainerStatus MayaAPIServiceOutputLabel = "openebs.io/controller-container-status"
```

Comment:
this should be of type `VolumeKey`
the name should be `ControllerContainerSatusVK`